README.md

```diff
@@ -1,140 +1,126 @@
+```
+Usage:
+  CSSS.exe -c | -o | -p | -s | [-h] | [-m]
+
+Examples:
+  CSSS.exe -c
+  CSSS.exe -o -m
+  CSSS.exe -p
+  CSSS.exe -s
+  CSSS.exe -h
+
+
+Required arguments (at least one is needed):
+  -c, --check:    Checks the config files for any problems
+  -o, --observe:  Observes CSSS running before preparing it (implies '-c')
+  -p, --prepare:  Prepares CSSS ready for image release (implies '-c')
+  -s, --start:    Starts the scoring system
+
+
+Optional arguments:
+  -h, --help:     Shows this help message
+
+
+Developer arguments (all optional):
+  -m, --multiple: Allows multiple instances of CSSS to run concurently
+```
+
+The required arguments have been chosen to spell 'COPS', as competitors are 'policing' the security of the computer.
+
+Under normal image building, the `-o` option should be passed, as this allows CSSS to run as it would for training, but without affecting the files used to list the 'issues'.
+
+When you are ready to release the image, __with administrative privileges__ (e.g. `sudo`, "Run as administrator") run CSSS with the `-p` option to prepare the necessary files and allow CSSS to start automatically on computer reboots.
+> :warning: Running CSSS with the `-p` argument will encrypt the issue files. Make sure that you have a snapshot of your image before you run this command, otherwise you will need to set everything up again should there be any problems.
+
+## Contributing
+Thanks for your interest in CSSS. Read more about [contributing to the project](CONTRIBUTING.md) to get started.
+Please note that this project is released with a [Contributor Code of Conduct](#CODE_OF_CONDUCT.md). By participating in this project you agree to abide by its terms.
+
+### NuGet Packages
+This project uses NuGet packages. Their project source code pages and licenses can be found below:
+* [HtmlAgilityPack](https://htmlagilitypack.codeplex.com/)
+* [Newtonsoft.Json](http://www.newtonsoft.com/json)
+* [NLog](http://nlog-project.org/)
+* [NUnit](https://www.nunit.org/)
+
+### Other Projects
+The following projects and source code are included in CSSS. Their licenses project pages can be found below:
+* [OSVersionInfo](https://www.codeproject.com/Articles/73000/Getting-Operating-System-Version-Info-Even-for-Win)
+
+## Copyright
+CyberPatriot is copyright of US Air Force Association
+
+CyberCentrurion is copyright of Cyber Security Challenge UK
+
+## Disclaimer
+This program is designed to score the state of a computer image against a pre-defined checklist of "issues". It is not designed to rate your computer security, provide security advice, or anything else that isn't related to the CyberPatriot / CyberCenturion competitions.
+
+## License Terms
+CSSS is publised under the GNU GPL v3 License, see the [LICENSE](LICENSE.md) file for more information.
```

Comment:
License information needs to be moved after "Contributing" / before "NuGet Packages" as per the table of contents.